src/components/PaginationFooter/PaginationFooter.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import { Select, Stack, Text, ChakraProvider } from '@chakra-ui/react';
+import { PropTypes } from 'prop-types';
+import { PNPBackend } from '../../utils/utils';
+import {
+  Pagination,
+  usePagination,
+  PaginationNext,
+  PaginationPrevious,
+  PaginationContainer,
+} from '@ajna/pagination';
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`@ajna/pagination` import should occur before import of `../../utils/utils`
```suggestion
import {
  Pagination,
  usePagination,
  PaginationNext,
  PaginationPrevious,
  PaginationContainer,
} from '@ajna/pagination';
import { PNPBackend } from '../../utils/utils';
```